Add in Hello World example across docs (#1608)
slackapi/bolt-js: docs/_future/getting_started_future.md

With your app running, access your workspace and paste the URL from the Trigger you created in the previous step into a message in a public channel.

> > 💡 The Trigger will automatically be saved as a channel bookmark under "Workflows" for easy access.

Reviewer: ```suggestion
> 💡 App Triggers are automatically be saved as a channel bookmark under "Workflows" for easy access.
```
